collector/stat_replication.go

```diff
@@ -83,17 +89,24 @@ func (c *statReplicationScraper) Scrape(ctx context.Context, conn *pgx.Conn, ver
 	var applicationName, state, syncState string
 	var clientAddr net.IP
 	var pgXlogLocationDiff float64
+	/* When querying pg_stat_replication it may be that we don't have
```

Comment:
@ahjmorton I didn't have time to go into details here, but could you share more details about these metrics disappearing?

Later you make a conditional have a value to report these metrics. We should avoid that, as metrics that disappear from Prometheus are complex to deal with 

https://prometheus.io/docs/practices/instrumentation/#avoid-missing-metrics

Reply:
> @ahjmorton I didn't have time to go into details here, but could you share more details about these metrics disappearing?

If the WAL sender process didn't have an activity for some period of time (not sure how long, might need to look at source) then Postgres would null out those "*_lag" fields.

> Later you make a conditional have a value to report these metrics. We should avoid that, as metrics that disappear from Prometheus are complex to deal with

I was trying to avoid reporting metrics for nodes that didn't have a WAL sender process however those would be eliminated by the query as they wouldn't have a row in `pg_stat_replication`.
Going to change this to default to zero in case of null.